test/specs.test.js

```diff
@@ -11,7 +11,9 @@ const clone = require('clone');
 
 const dir = path.resolve(__dirname, 'specs'),
     files = fs.readdirSync(dir).filter(function (f) {
-        return f.match(/\.yaml$/);
+        const isYaml = f.match(/\.yaml$/);
+        const grep = process.env.GREP ? (new RegExp(process.env.GREP)).test(f) : true;
```

Comment:
@wyze nice, I hadn't thought of trying this as the assertions were being generated programmatically; works like a charm.